src/cli/utils/fixtures/asyncDataWithAsyncFunctionAndAsyncParameter.js

```diff
@@ -0,0 +1,27 @@
+		async function password(){
+
+			return Promise.resolve("test");
+		}
+
+		async function asyncConfig(){
```

Comment:
This is not necessary, you can just return the config in the export

Reply:
i don't understand this

Reply:
Why don't you just export the config instead of having an intermediary method? We don't need to have a method that return the config and then a method that await it and then export the last method.
We can just have

async function password()...
module.exports = { ... }

Reply:
done